controlplane/kubeadm/internal/workload_cluster.go

```diff
@@ -394,6 +394,47 @@ func (c *Cluster) ReconcileKubeletRBACRole(ctx context.Context, version semver.V
 	return nil
 }
 
+// ForwardEtcdLeadship forward etcd leadship to first follower
+func (c *Cluster) ForwardEtcdLeadship(ctx context.Context, machine *clusterv1.Machine) error {
+	if machine == nil || machine.Status.NodeRef == nil {
+		// Nothing to do, no node for Machine
+		return nil
+	}
+
+	return c.forwardEtcdLeadship(ctx, machine.Status.NodeRef.Name)
+}
+
+func (c *Cluster) forwardEtcdLeadship(ctx context.Context, nodeName string) error {
```

Comment:
what was the intention behind splitting this private method out from the `FowardEtcdLeadship` method?